tests/unit/jobserver/management/commands/test_cancel_jobs.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management import call_command
+
+
+_FAKE_ARGS = ("abcde1234", "action1", "action2")
+
+
+def test_command(monkeypatch, log_output):
+    test_response_body = b'{"foo": "bar"}'
+
+    monkeypatch.setattr(
+        "jobserver.rap_api.cancel",
+        lambda job_request_id, actions: test_response_body,
+    )
+
+    call_command("cancel_jobs", *_FAKE_ARGS)
+
+    assert log_output.entries[0] == {
+        "event": test_response_body,
+        "log_level": "info",
+    }
```

Comment:
this test does so little that I'm /almost/ tempting to replace it with `pragma: nocover`.... we should keep it though.